src/commands/hug.js

```diff
@@ -1,8 +1,66 @@
+import axios from 'axios';
+import Discord from 'discord.js';
 import DiscordCommand from './command.js';
 
 export default class HugCommand extends DiscordCommand {
-    async execute() {
-        const hug = this.getEmojiByName('HugSpin');
-        this.message.channel.send(`${hug}`);
+
+  async execute() {
+    if (!this.member) {
+      this.send("You must be in a Roo Bot enabled Discord server to use this command");
```

Comment:
This was actually my bad, I set a bad precedent.

anytime we use `this.send(...)` we should be using `await this.send(...)` to ensure it executes and throws correctly